File: withgoogle/CodeJam/2019/Qualification/YourOwnWay/MirrorPathConstruction/test/SolutionTests.java
import static org.junit.Assert.*;

import java.io.*;

import org.junit.*;

public class SolutionTests {

    @Test
    public void test0() throws FileNotFoundException {
        InputReader in = new InputReader(new FileInputStream("test_data/0.in"));
        Solution solution = new Solution(in);
        String[] result = solution.solve();
        assertArrayEquals(new String[] {
            "Case #1: ES",
            "Case #2: SSEEESES"
        }, result);
    }

}

File: withgoogle/KickStart/2019/Round B/Building Palindromes/AdHocCounting/Solution.Tests/SolutionTests.cs
using System;
using System.IO;
using NUnit.Framework;

public class SolutionTests {

	[Test]
	public void Test0() {
		Solution solution = new Solution(new StreamReader("../../../../test_data/0.in"));
		string[] result = solution.Solve();
		Assert.AreEqual(new string[] {
			"Case #1: 3",
			"Case #2: 0",
		}, result);
	}

}
